internal/config/config.go

```diff
@@ -426,6 +426,34 @@ func (c Config) DelFrom(r io.Reader) error {
 	return scanner.Err()
 }
 
+// ContextKeyString is type(string) for contextKey
+type ContextKeyString string
+
+// ContextKeyForTargetFromConfig - key for context for target from config
+const ContextKeyForTargetFromConfig = ContextKeyString("ContextKeyForTargetFromConfig")
+
+// ParseConfigTargetID - read all targetIDs from reader
+func ParseConfigTargetID(r io.Reader) (ids map[string]bool, err error) {
```

Comment:
I don't think SetConfigKV API will accept multiple targets. It works with only single target. We don't need this parser.

Reply:
@harshavardhana help there too.